app/javascript/components/widgets/events/panel.jsx

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import styled from 'styled-components';
+import { colors, fontSizes, fonts, spacing } from '@lib/theme';
+import StarIcon from '@assets/images/icons/icon-star.svg';
+import GiftIcon from '@assets/images/icons/icon-gift.svg';
+import UsersIcon from '@assets/images/icons/icon-users.svg';
+import UserPlusIcon from '@assets/images/icons/icon-user-plus.svg';
+import { Row } from '@components/row';
+import { WhiteSubTitle, WhiteTitleLarge } from '@components/typography';
+
+const Title = styled.div`
+  margin-bottom: 72px;
+  margin-left: 150px;
+  width: 45vw;
+`;
+
+const Heading = styled.div`
+  font-size: ${fontSizes.large};
+  font-family: ${fonts.regular};
+  margin-bottom: ${spacing.m};
+  color: ${p => p.textColor};
+`;
+
+const Column = styled.div`
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+`;
+
+const Number = styled.div`
+  color: #ffffff;
+  font-size: 136px;
+  font-weight: 500;
+  font-family: GT America-Extended;
+`;
+
+const Icon = styled.div`
+  margin-right: ${spacing.l};
+  margin-top: ${spacing.s};
+`;
+
+const Grid = styled.div`
+  width: 45vw;
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-template-rows: repeat(2, 1fr);
+  margin-left: 150px;
+`;
+
+const GridItem = styled.div`
+  display: flex;
+  width: 435px;
+  height: 375px;
+`;
+
+const Anniversaries = () => {
+  return (
+    <Column>
+      <Number>0</Number>
+      <Row>
+        <Icon>
+          <img src={StarIcon} alt="star" width={24} height={32} />
+        </Icon>
+        <Heading textColor={colors.yellow}>Anniversaries this week</Heading>
+      </Row>
+      <WhiteSubTitle>No Mojo Anniversaries</WhiteSubTitle>
+    </Column>
+  );
+};
+
+const Birthdays = () => {
+  return (
+    <Column>
+      <Number>0</Number>
+      <Row>
```

Comment:
NIt: we're in Grid -> GridItem -> Column -> Row, but there is only ever one column and one row in a GridItem so its a little misleading (in terms of naming). The Column with flex and height 100% feels like a css hack to fill. I'd be suspicious of the grid definition if these weren't sizing correctly.

Reply:
Forgot to link this which may help clarify flex/flex-basis vs height/width: https://mastery.games/post/the-difference-between-width-and-flex-basis/

Reply:
You're right, I removed the Column because it was not needed. The row is used to keep the icon inline with its heading text so I think we should keep that. What do you think?